Test/Utils/ContrailAPI/Global.ps1

```diff
@@ -0,0 +1,16 @@
+function FQNameToUuid {
```

Comment:
This function is used in only one file, therefore why is it in "Global.ps1" file? It's like the most local thing I've ever seen :)

We should probably move it to file where it's used and think about extracting it to separate file when another client of this function is needed OR rename this file to `FQName.ps1` or similar.

Reply:
It is global from the API level. It can be used for all the resources.  If somebody will be writing another API fragment, he will not be searching all the files to check if somebody wrote some functions like this. Maybe we can name this file "UniversalFunctions" or something like that?

Reply:
Ooooh, so that's why it's called Globals.ps1. If that's the reason then it's fine.